### PR TITLE
feat: default configs

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

**Review by netanelC — CHANGES_REQUESTED** [review]
The checks do not validate  the extension of the config file (for example: "jso" instead of "json"), only the schema.

File: README.md [hunk 1]
```diff
@@ -138,6 +139,33 @@ The script contains the following validations:
...
+  }
+]
+```
+
+### Validations
+
+The following validations are performed on the default config files:
+
+1. Checks that there is a schema for each config file.
+2. Checks that the configs file is valid.
```
[the remark this answers is not in the document]
Comment (CptSchnitz) [reply README.md:164]: there are multiple configs in the file..

File: scripts/build.mts [hunk 2]
```diff
@@ -35,15 +35,25 @@ for await (const file of filesTreeGenerator(schemasFolder)) {
...
+  //     break;
+  //   case '.json':
+  //     if (file.name.endsWith('.schema.json')) {
+  //       content = JSON.parse(await fsPromise.readFile(fullPath, 'utf-8'));
+  //       break;
+  //     }
+  //     continue;
+  //   default:
+  //     continue;
+  // }
```
[the remark this answers is not in the document]
Comment (CptSchnitz) [reply scripts/build.mts:50]: no

File: scripts/validate/validate.mts
[the remark this answers is not in the document]
Comment (CptSchnitz) [reply scripts/validate/validate.mts:1]: its literally what it does, its the main entrypoint to all validations, just its less recommended to call file index.

**Comment by CptSchnitz** [comment]
fixed not checking file extensions for the configs.

File: scripts/validate/validate.mts
[the remark this answers is not in the document]
Comment (netanelC) [reply scripts/validate/validate.mts:1]: OK

File: scripts/build.mts (hunk 2 above)
[the remark this answers is not in the document]
Comment (netanelC) [reply scripts/build.mts:50]: yes. Oy Vaavoy

**Review by netanelC — APPROVED** [review]

**Pull request merged**